daemon/cmd/fqdn_test.go

```diff
@@ -37,13 +39,32 @@ type DaemonFQDNSuite struct {
 	d *Daemon
 }
 
+var notifyOnDNSMsgBenchSetup sync.Once
+
 func setupDaemonFQDNSuite(tb testing.TB) *DaemonFQDNSuite {
 	testutils.IntegrationTest(tb)
 
-	re.InitRegexCompileLRU(defaults.FQDNRegexCompileLRUSize)
+	// We rely on a sync.Once to complete the benchmark setup that initialize
+	// read-only global status.
+	// Also, node.SetTestLocalNodeStore() panics if it called more than once.
+	notifyOnDNSMsgBenchSetup.Do(func() {
+		// set FQDN related options to defaults in order to avoid a flood of warnings
+		option.Config.DNSProxyLockCount = 131
```

Comment:
Should we set these values as constants? Or better can the default can be configured somewhere else in the field definition?